### ISSUE TITLE
Better tools needed to determine the rank when approving a kata

### ISSUE THREAD

**Issue by Blind4Basics** [Codewars/codewars.com#1939]
Hi,

Currently, there are 3 available informations when choosing the rank:

* author estimation: counts for 1 vote, totally isolated from the 2 other data
* users avg: ceiled value
* PU avg: not entirely sure about this one... I believe it's rounded?

problem is:

With "some" users (namely, ZED. Essentially...) who are litterally trolling the rank suggestions, those 3 datas often lead these days to a "what the hell should I do?" because those values do not mean anything anymore because of the roundings.

Suggested solution :

A way to see the partition of the votes would allow a pretty good idea about what to do (histogram: number of votes against kyu rank). 
Possible implementation : a link in the edit panel that would lead to another page with the related diagrams.

cheers

**Comment by Steffan153** [comment]
Good idea. On some katas I've approved so far, it took me a long time to make up my mind what to rank it :confused:

**Comment by Blind4Basics** [comment]
(note: the histograms are better than "not rounded values", which would still give a lot of weight to "free electrons votes")

**Comment by dinglemouse2250** [comment]
FYI - here's an idea I suggested a long time ago #1247 which is kind of in this same area...

**Comment by Voileexperiments** [comment]
But first, the ranks that can be approved should not be *locked* by author estimated rank and the lower of user average/power user average, because the people who can approve them either obeys them or ignores them entirely anyway. Also it's only enforced at the front-end - sending values not available to the back-end during approval will still work as expected.

**Comment by Blind4Basics** [comment]
![image](https://user-images.githubusercontent.com/32236948/65822895-1b9a4e00-e24c-11e9-9119-e6479074b84d.png)

I guess that's you, Voile?

**Comment by Voileexperiments** [comment]
> I guess that's you, Voile?

No idea, why don't you check the revisions page instead?

**Comment by Blind4Basics** [comment]
right...! x)

**Comment by hobovsky** [comment]
Update: rank votes can be now checked [with API](https://www.codewars.com/api/v1/code-challenges/57acc8c3e298a7ae4e0007e3/assessed-ranks). There is no UI for this in the platform, but can be done with [user script](https://github.com/hobovsky/polyglot?tab=readme-ov-file#what-features-does-it-provide):

![Image](https://github.com/user-attachments/assets/b5863e94-7daa-49d5-9198-a3e02cff1965)
